tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -6200,7 +6202,7 @@ Status ConvertSquaredDifference(OpConverterParams* params) {
   return Status::OK();
 }
 
-#if IS_TRT_VERSION_GE(8, 2, 1, 6) || defined(TF_TRT_USE_EFFICIENT_NMS_PLUGIN)
+#if IS_TRT_VERSION_GE(7, 1, 3, 0)
```

Comment:
I'm not sure if we support TRT 6 still, I believe not, but perhaps 7.0 is. But in any case, there's several other if-gates for 7.1 throughout the converter code still, so perhaps everything should be tested for < 7.1 compat at some point? (I don't have enough bandwidth to do so though).

Reply:
So I went through the release notes of 7.1.0 and 7.1.3 and it seems there are no API changes between 7.0 and 7.1.x that would impact the plugin build or anything related to the NMS converter. So in theory it might work for 7.0, but testing for that version will have to be done on a separate PR regardless, as I don't currently have a good way to test with 7.0 (because it's compatible with CUDA 10 only).